sites/partners/styles/overrides.scss

```diff
@@ -197,3 +197,17 @@
   vertical-align: 15%;
   margin-inline-start: var(--seeds-s3);
 }
+
+// Temporary short-term overrides for contrast compliance
+.darker-link {
+  --seeds-color-primary: var(--seeds-color-blue-700);
+  --bloom-color-primary: var(--seeds-color-blue-700);
+}
+
+.darker-alert {
+  --seeds-color-primary: var(--seeds-color-red-700);
+}
+
+.darker-help-text {
+  --field-value-help-text-color: var(--seeds-input-text-label-color);
+}
```

Comment:
question: Will this require specific changes in any of the jurisdictions? I know that we are planning to keep links blue everywhere, but don't know if Doorway has edited any of these colors

Reply:
Nope it shouldn't! Ideally we'd be able to set the links to the link color instead of blue, but we don't have that token yet, and jurisdictions haven't customized the alert or help text colors